pyramid_storage/gcloud.py

```diff
@@ -43,30 +46,49 @@ def from_settings(cls, settings, prefix):
             ('gcloud.credentials', False, None),
             ('gcloud.project', False, None),
             ('gcloud.bucket_name', True, None),
-            ('gcloud.acl', False, DEFAULT_FILE_ACL),
+            ('gcloud.acl', False, None),
             ('base_url', False, ''),
             ('extensions', False, 'default'),
             # Gcloud Connection options.
             ('gcloud.auto_create_bucket', False, False),
-            ('gcloud.auto_create_acl', False, DEFAULT_BUCKET_ACL),
+            ('gcloud.auto_create_acl', False, None),
             ('gcloud.cache_control', False, None),
+            ('gcloud.uniform_bucket_level_access', False, False),
         )
         kwargs = utils.read_settings(settings, options, prefix)
         kwargs = dict([(k.replace('gcloud.', ''), v) for k, v in kwargs.items()])
         return cls(**kwargs)
 
     def __init__(self, credentials, bucket_name, project=None, acl=None, base_url='',
                  extensions='default', auto_create_bucket=False,
-                 auto_create_acl=DEFAULT_BUCKET_ACL, cache_control=None):
+                 auto_create_acl=None, cache_control=None,
+                 uniform_bucket_level_access=False):
+        self.logger = logging.getLogger(__name__)
```

Comment:
since `logger` does not need a reference to `self`, we define it as a module global instead of a class member